src/app/reporting/reporting.cpp

```diff
@@ -39,8 +40,8 @@ void MatterReportingAttributeChangeCallback(const ConcreteAttributePath & aPath)
     // applications notifying about changes from their end.
     assertChipStackLockedByCurrentThread();
 
-    emberAfAttributeChanged(aPath.mEndpointId, aPath.mClusterId, aPath.mAttributeId,
```

Comment:
These should indeed be updated out: libCHIP should not bind into ember. I think these changes independently are actually correct, however we have to pipe then through something else than `MatterRerportingAttributeChangeCallback` ... the callback assumes too many globals and data ownership differences.

Reply:
We could maybe mark this as temporary and have `TermpoaryReportAttributeChanged` instead of `ReportAttributeChanged` and a specific TODO to not use this anymore (please add a bug for this ... I can add some interfaces to AAI/CHI to allow context usage to report attribute changes).

Reply:
Added the bug https://github.com/project-chip/connectedhomeip/issues/36381